src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] garage = {new Bulldozer(),
```

Comment:
its better to have separate line in that case
```suggestion
        Machine[] garage = {
            new Bulldozer(),
```